gammapy/detect/test_statistics.py

```diff
@@ -35,15 +38,33 @@ class TSMapResult(Bunch):
 
     Attributes
     ----------
-    ts : `~numpy.ndarray`
+    ts : ndarray
         Estimated TS map
-    amplitude : `~numpy.ndarray`
+    amplitude : ndarray
         Estimated best fit flux amplitude map
-    niter : `~numpy.ndarray`
+    niter : ndarray
         Number of iterations map
     runtime : float
         Time needed to compute TS map.
     """
+    def read(self, filename):
```

Comment:
If possible I'd like `read` to always be a `@staticmethod` like `Table.read` in astropy or [EnergyDependentTablePSF.read](https://gammapy.readthedocs.org/en/latest/_modules/gammapy/irf/psf_table.html#EnergyDependentTablePSF.read) in Gammapy.